### ISSUE TITLE
Need examples

### ISSUE THREAD

**Issue by wschin** [dotnet/docs#8253]
It's unlikely users can implement things like https://stackoverflow.com/questions/52190423/c-sharp-access-unmanaged-array-using-memoryt-or-arraysegmentt with this doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a5bf7ff4-3fab-4cda-7943-5717376a595f
* Version Independent ID: 721ccd28-5c40-826f-5419-4afe560ebfc6
* Content: [MemoryManager&lt;T&gt; Class (System.Buffers)](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.memorymanager-1?view=netcore-2.1)
* Content Source: [xml/System.Buffers/MemoryManager`1.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Buffers/MemoryManager`1.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
You're right @wschin. We haven't documented all the .NET Core APIs yet. We'll take care of this on a future sprint.
